### PR TITLE
SPC 835 Docstring bug in single line and trailing spaces

### PR DESCRIPTION
**Pull request by Jarelk**

### CODE REVIEW

File: utility/comment_checker.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class CommentChecker(BaseChecker):
 
     The following linting errors get caught:
         comment-should-have-one-space: Comment should start with only one space.
-        comment-missing-period: Comment has to end with a period.
+        comment-missing-period: Comment has to end with a dot.
```
Comment (saberzero1) [inline utility/comment_checker.py:18]: Why is this changed back to a dot?

**Review by saberzero1 — CHANGES_REQUESTED** [review]

File: utility/comment_checker.py (hunk 1 above)
Comment (Jarelk) [reply utility/comment_checker.py:18]: Reeeee ik verander hem weer naar period.

**Review by saberzero1 — APPROVED** [review]

**Review by Li-Yeun — APPROVED** [review]
